### ISSUE TITLE
Static check on cron expression

### ISSUE THREAD

**Issue by jianghoy** [aws-cloudformation/cfn-python-lint#1723]
*cfn-lint version: (0.35.1)*

*Description of issue.*
cfn-lint didn't detect malformed cron expression, e.g.: 
```yml
AWSScheduledEvent
    Type: AWS::Events::Rule
    Condition: IsProdNA
    Properties:
      Description: Event that triggers my_awesome_lambda daily.
      Name: MyAwesomeLambdaTrigger
      ScheduleExpression: "cron(0 0 0 ? ? *)"
```
The cron expression is wrong and got failure:
```
Parameter ScheduleExpression is not valid. (Service: AmazonCloudWatchEvents; Status Code: 400; Error Code: ValidationException; Request ID: ...; Proxy: null)
```
Please provide as much information as possible:
* Feature request:
  * Add static checking on cron expression. Because AWS scheduled event doesn't support second level specification, it's actually different that other cron by missing the second, It would be nice to have static check beforehand.

Cfn-lint uses the [CloudFormation Resource Specifications](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) as the base to do validation. These files are included as part of the application version. Please update to the latest version of `cfn-lint` or update the spec files manually (`cfn-lint -u`)

**Comment by PatMyron** [comment]
duplicate of https://github.com/aws-cloudformation/cfn-python-lint/issues/816
